data/Language/C.yaml

```diff
@@ -0,0 +1,76 @@
+identifier: C
+full_name: C
+wikidata: Q15777
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/c/C.g4
+aliases:
+  - ANSI C
+  - ISO C
+  - Standard C
+extensions:
+  - c
+  - h
+line_continuation: '\'
+delimiters:
+  # comment delimiters
+  - double_slash
+  - multiline_slash_star
+  # string delimiters:
+  - c_char
+  - double_quote_slash_escape
+keywords:
+  - auto
+  - break
+  - case
+  - char
+  - const
+  - continue
+  - default
+  - do
+  - double
+  - else
+  - enum
+  - extern
+  - float
+  - for
+  - goto
+  - if
+  - int
+  - long
+  - register
+  - return
+  - short
+  - signed
+  - sizeof
+  - static
+  - struct
+  - switch
+  - typedef
+  - union
+  - unsigned
+  - void
+  - volatile
+  - while
+  - #include
```

Comment:
needs quotes around it, otherwise the `#` starts a yaml comment

Reply:
Done